cli/agent.go

```diff
@@ -197,6 +198,7 @@ func (r *RootCmd) workspaceAgent() *clibase.Cmd {
 				return xerrors.Errorf("add executable to $PATH: %w", err)
 			}
 
+			subsystem := inv.Environ.Get(agent.EnvAgentSubsystem)
```

Comment:
Is there a benefit to just getting the env manually instead of via `cmd.Options` where the other flags are?

Reply:
It's not a command option it's just an env var that may or may not be there depending on where the agent is being run. I used `inv.Environ` in lieu of `os.Getenv` so I could test it.